### ISSUE TITLE
using vis as editor

### ISSUE THREAD

**Issue by Docbroke** [leo-arch/clifm#62]
I did add vis to mimelist.cfm as
```
X:(^text/.*|application/json|inode/x-empty)=vis;micro;nano;vim;vi;emacs;ed;leafpad;mousepad;kate;gedit;pluma
```
This makes clifm wait on empty prompt. This issue is not present with vi, which seems to load alright.
I can start vis normally if I run it from clifm command prompt (`vis <some_file>`), but it doesn't work with Lira.

**Comment by leo-arch** [comment]
Hi @Docbroke! I'll take a look at it. Thanks for reporting this bug.

**Comment by leo-arch** [comment]
Well, it seems that, for some reason, `vis` doesn't like when you silence STDERR. I don't know why it behaves like this, but as far as I know it is the only program refusing to run when you redirect standard error. Indeed, try running `vis` on any other shell while silencing STDERR:

```sh
vis 2>/dev/null
```

It hangs.

I assume therefore that this issue belongs to `vis` itself, and not to CliFM.

Btw, the manpage does not mention STDERR at all, so that it doesn't seem to be an intended behavior.

**Comment by Docbroke** [comment]
Thanks for looking into this. I will report this to vis.
I have noticed another issue with clifm, when I try to run clifm from script, the process gets stopped.
For example,
```
#!/bin/sh
clifm
```
when this file is executed, clifm print file list and it is stopped,
```
<file list>
[1]+ stopped
```

**Comment by leo-arch** [comment]
I was already aware of that. It's in the TODO list.

**Comment by leo-arch** [comment]
Hi @Docbroke. _CliFM_ should be able now to be launched via a script.
